software/clipbucket.yml

```
@@ -0,0 +1,12 @@
+name: "ClipBucket"
+website_url: "https://clipbucket.oxygenz.fr"
```

```suggestion
website_url: "https://clipbucket.fr/"
```

Most confusing pull request review ever, right here :joy: sorry